go/host/host.go

```diff
@@ -545,7 +546,7 @@ func (h *host) publishRollup(producedRollup *common.ExtRollup) {
 	tx := &ethadapter.L1RollupTx{
 		Rollup: encodedRollup,
 	}
-	h.logger.Info(fmt.Sprintf("Publishing rollup with height=%d, size=%dKB", producedRollup.Header.Number, len(encodedRollup)/1024))
+	h.logger.Info("Publishing rollup", "height", producedRollup.Header.Number, "size", len(encodedRollup)/1024, "hash", producedRollup.Hash())
```

Comment:
You can use `RollupHashKey` and `RollupHeightKey` constants for these param keys so that they're standardised